supported-devices/template.md

```diff
@@ -40,7 +40,7 @@ If you're on a smaller screen, scroll sideways to see whole table.
 :::
 
 <AddScript script-url="../../supported-devices.js"/>
-<p>Search: <input type="text" class="deviceSearch"></p>
+<p><input type="text" class="deviceSearch" placeholder="Enter Your Boardname Or Device Name"></p>
```

Comment:
removing the 'Search' prefix, coupled with the css color changes, makes it almost impossible to tell that there is a search bar there now:

![image](https://github.com/chrultrabook/docs/assets/948902/7cea069b-5788-4859-ba4f-8ace10970aca)

the old version was much better:
![image](https://github.com/chrultrabook/docs/assets/948902/530e1881-abe0-41d2-bd03-1ec71d3e067d)

Reply:
I can change the bg and make it pop more

Reply:
it's one example of a larger problem IMO. I don't know what issue this PR is trying to solve

Reply:
The field and text are a accessibility issue aswell due to missing contrast